cockatrice/src/deckview.cpp

```diff
@@ -1,5 +1,6 @@
 #include <QApplication>
 #include <QGraphicsSceneMouseEvent>
+#include <QDebug>
```

Comment:
Is `QDebug` needed in the PR?
I see that it outputs `"LEFT"`, though I assume this was for your own testing.